src/Sarif/Sarif.csproj

```diff
@@ -41,8 +41,8 @@
     <!--
     Marking these items as content causes them to be included in the NuGet package.
     -->
-    <Content Include="Schemata\sarif-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
-    <Content Include="Schemata\sarif-external-property-file-$(SchemaVersionAsPublishedToSchemaStoreOrg).json" />
+    <Content Include="Schemata\sarif-2.1.0-rtm.5.json" />
+    <Content Include="Schemata\sarif-external-property-file-2.1.0-rtm.5.json" />
```

Comment:
>sarif-external-property-file-2.1.0-rtm.5.json [](start = 31, length = 45)

I don't know why the changes to these lines happened -- they happened for me as well in my PR -- but please revert these two lines to continue using the symbol `$(SchemaVersionAsPublishedToSchemaStoreOrg)`.

Reply:
File-based search and replace is what did it to me.

---
In reply to: [382834618](https://github.com/microsoft/sarif-sdk/pull/1789#discussion_r382834618) [](ancestors = 382834618)